### PR TITLE
[12.0][FIX] - hr_attendance_rfid - Handle ValidationError properly

### PR DESCRIPTION
**Pull request by lubusax**
A ValidationError has to be converted to a string explicitly, otherwise an error is produced if it is used as a string.

### CODE REVIEW

**Comment by lubusax** [comment]
ping @AaronHForgeFlow

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM.  First then get this one merged and then we can care of versions 13.0 to 15.0

**Comment by lubusax** [comment]
> 

Thx Aaron, 14 is already merged. 15 is waiting for approval. 13 has low codecov/project (95,2%; -0,06%), so I will write additional testing to reach the minimum.

**Comment by AaronHForgeFlow** [comment]
@lubusax , ok then it should be merged at least in v15. It is priority to keep branches even, specially newest versions.

@OCA/human-resources-maintainers this is a backport of https://github.com/OCA/hr-attendance/pull/128

**Pull request merged**
